BitFlow/Optimization.py

```diff
@@ -19,11 +19,11 @@ def handleIB(self, node):
         ib = 0
         x = self.node_values[node]
         if isinstance(x, Interval):
-            alpha = 2 if (log2(x.hi) % 1 == 0) else 1
+            alpha = 2 if (log2(abs(x.hi)) % 1 == 0) else 1
```

Comment:
What is this %1 ? If you are checking if the floating point value is an integer, you can use val.is_integer().

Another (hacky) trick to get the number of bits of an integer is to do len(bin(val))-2

Reply:
I blindly applied the equation in section five of the paper; i'll edit it to use is_integer()